frontend/src/views/SubmitView.js

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import { StateContext } from "../State";
+
+const SubmitView = () => {
+	const context = React.useContext(StateContext);
+	const [ componentState, setComponentState ] = React.useState({
+		groupID: 1,
+		categoryID: 1,
+		title: "",
+		description: "",
+		submittedBy: context.user.id
+	});
+
+	const handleSubmitTicket = async () => {
+		var response = await fetch(`http://localhost:8080/tickets`, {
+			method: "POST",
+			headers: {
+                "Accept": "application/json",
+                "Content-Type": "application/json"
+            },
+			body: JSON.stringify(componentState)
+		});
+
+		var data = await response.json();
+
+		// TODO: Show an alert with ticket submission status
+	}
+
+	const handleGroupSelected = (event) => {
+		setComponentState({
+			...componentState,
+			groupID: Number(event.target.value)
+		});
+	}
+
+	const handleCategorySelected = (event) => {
+		setComponentState({
+			...componentState,
+			categoryID: Number(event.target.value)
+		});
+	}
+
+	const handleChange = (event) => {
+		setComponentState({
+			...componentState,
+			[event.target.name]: event.target.value
+		});
+	}
+
+	return (
```

Comment:
Thought Experiment - the code returned here from this functional component is starting to get a little on the lengthy and complicated side.  It might be worth splitting this down a bit more into subcomponents - usually when a React component gets to the point of exceeding 100 lines of code, I start to consider if there is anything that could be subdivided further for clarity's sake.  The container, for instance, may be separable from the form within - the container could be its own component with a form within it, using context or props to perpetuate the methods that are located in this file.